cookbook-react-native/recipes/pushNotification/readme.md

```diff
@@ -0,0 +1,358 @@
+
+```xml
+ <uses-permission android:name="android.permission.VIBRATE" />
+    <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
+<!-- ADDED with this meta-data we can modify the icon in notifications, remember change resource name and apply the corrects sizes  -->
+    <meta-data  android:name="com.google.firebase.messaging.default_notification_icon" android:resource="@mipmap/ic_notification" />
+
+
+<!-- Change the value to true to enable pop-up for in foreground on receiving remote notifications (for prevent duplicating while showing local notifications set this to false) -->
+        <meta-data  android:name="com.dieam.reactnativepushnotification.notification_foreground"
+                    android:value="false"/>
+        <!-- Change the resource name to your App's accent color - or any other color you want -->
+        <meta-data  android:name="com.dieam.reactnativepushnotification.notification_color"
+                    android:resource="@color/white"/> <!-- or @android:color/{name} to use a standard color -->
+
+        <receiver android:name="com.dieam.reactnativepushnotification.modules.RNPushNotificationActions" />
+        <receiver android:name="com.dieam.reactnativepushnotification.modules.RNPushNotificationPublisher" />
+        <receiver android:name="com.dieam.reactnativepushnotification.modules.RNPushNotificationBootEventReceiver">
+            <intent-filter>
+                <action android:name="android.intent.action.BOOT_COMPLETED" />
+                <action android:name="android.intent.action.QUICKBOOT_POWERON" />
+                <action android:name="com.htc.intent.action.QUICKBOOT_POWERON"/>
+            </intent-filter>
+        </receiver>
+
+        <service
+            android:name="com.dieam.reactnativepushnotification.modules.RNPushNotificationListenerService"
+            android:exported="false" >
+            <intent-filter>
+                <action android:name="com.google.firebase.MESSAGING_EVENT" />
+            </intent-filter>
+        </service>
+
+```
+
+In `android/app/src/main/res/values/colors.xml` (Create the file if it doesn't exist) for the notification_color
+
+```xml
+<resources>
+  <color name="white">#FFF</color>
+</resources>
+```
+
+If you need remote Notifications:
+
+`In android/build.gradle`
+
+```gradle
+buildscript {
+    ...
+    repositories {
+    // Check that you have the following line (if not, add it):
+    google()  // Google's Maven repository
+    }
+    dependencies {
+        ...
+        classpath 'com.google.gms:google-services:4.3.10'
+        ...
+    }
+}
+allprojects {
+  ...
+  repositories {
+    // Check that you have the following line (if not, add it):
+    google()  // Google's Maven repository
+    ...
+  }
+}
+```
+
+In `android/app/build.gradle`
+
+```gradle
+apply plugin: 'com.google.gms.google-services'
+
+dependencies{
+  implementation platform('com.google.firebase:firebase-bom:29.0.0')
```

Comment:
are you sure about this one? because usually you add the core or messaging lib here

Reply:
Not much, but when I added the SKD, Firebase suggest me added this line like this:
![Captura de Pantalla 2021-11-25 a la(s) 17 06 35](https://user-images.githubusercontent.com/54692138/143495857-0371f5be-38e2-4ad2-b2aa-712b5fc068fc.png)